### PR TITLE
Fixes flickering bug in Drawing tool Zoom Mode

### PR DESCRIPTION
**Pull request by vmartos**

### CODE REVIEW

**Comment by dfernandezramos** [comment]
:+1:

File: VAS.Drawing/SelectionCanvas.cs [hunk 1]
```diff
@@ -400,7 +400,7 @@ protected virtual void HandleMotionEvent (Point coords)
 			} else {
 				CursorMoved (userCoords);
 			}
-			MoveStart = userCoords;
```
Comment (ylatuya) [inline VAS.Drawing/SelectionCanvas.cs]: How is this fixing the issue if at the beginning of the function 	userCoords = ToUserCoords (coords); ?
If it happens that something is editing the userCoords un CursorMoved or sel.Drawable.Move, it's cheaper to userCoord.Copy () than to calculate again ToUserCoords (coords);

Comment (vmartos) [inline VAS.Drawing/SelectionCanvas.cs]: the CursorMoved is applying different RegionOfInterest in zoom mode, but not changing the usercords

Comment (ylatuya) [inline VAS.Drawing/SelectionCanvas.cs]: Got it now! Can you add comments to explain it please

Comment (ylatuya) [inline VAS.Drawing/SelectionCanvas.cs]: Got it, add a comment explaining that

**Pull request merged**